pkgs/tools/misc/xsel/default.nix

```diff
@@ -1,13 +1,22 @@
-{stdenv, fetchurl, xlibsWrapper}:
+{stdenv, fetchFromGitHub, xlibsWrapper, autoreconfHook }:
 
-stdenv.mkDerivation {
-  name = "xsel-1.2.0";
-  src = fetchurl {
-    url = http://www.vergenet.net/~conrad/software/xsel/download/xsel-1.2.0.tar.gz;
-    sha256 = "070lbcpw77j143jrbkh0y1v10ppn1jwmjf92800w7x42vh4cw9xr";
+stdenv.mkDerivation rec {
+  name = "xsel-${version}";
+
+  version = "git-2016-09-02";
```

Comment:
Ah, good point; I was just following the precedent I've seen elsewhere:

    ~/src/nixpkgs (nixos) λ ag 'git-\d\d\d\d-\d\d-\d\d' | wc -l
    59
    
    ~/src/nixpkgs (nixos) λ ag '\d\d\d\d-\d\d-\d\d-git' | wc -l
    2